index.html

```diff
@@ -53,5 +53,142 @@ <h1 id="myName">
+                <h2 class="card-title">Profesional Art Printing Data</h2>
+                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul class="card-techskills">
+                  <li class="card-skillset">HTML</li>
+                  <li class="card-skillset">Bootstrap</li>
+                  <li class="card-skillset">RUBY</li>
+                </ul>
+                <button class="card-btn">See project</button>
+              </div>
+              <div class="card-works">
+                <h2 class="card-title">Profesional Art Printing Data</h2>
+                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul class="card-techskills">
+                  <li class="card-skillset">HTML</li>
+                  <li class="card-skillset">Bootstrap</li>
+                  <li class="card-skillset">RUBY</li>
+                </ul>
+                <button class="card-btn">See project</button>
+              </div>
+            </div>
+        </div>
+      </section>
+      <section class="about">
+        <div class="about-me">
+          <h2 class="about-title">About me</h2>
+          <p class="about-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+          <button class="resume">Get my resume</button>
+        </div>
+         <div class="about-img"><img src="img/Frame.png"> </div> 
+         <hr class="divider">
+      </section>
+      <section class="profession-tools">
+
+          <div class="languages">
+              <div class="toolsTitle"> 
+                <img id="titles" src="img/Rectangle 55.png" alt="Languages">
+                <h1 >Language</h1>
+              </div>
+              <ul class="tools-details">
+                <li class="detail">JavaScript</li>
+                <li class="detail">Ruby</li>
+                <li class="detail">HTML</li>
+                <li class="detail">CSS</li>
+              </ul>
+          </div>
+
+          <div class="frameworks">
+            <div class="toolsTitle"> 
+              <img id="titles" src="img/Rectangle 56.png" alt="Languages">
+              <h1>Framework</h1>
+            </div>
+            <ul class="tools-details">
+              <li class="detail">React. js</li>
+              <li class="detail">Ruby on rails</li>
+              <li class="detail">RSpec</li>
+              <li class="detail">Capybara</li>
+            </ul>
+        </div>
+
+        <div class="skills">
+          <div class="toolsTitle"> 
+            <img id="titles" src="img/Rectangle 57.png" alt="Languages">
+            <h1>Skills</h1>
+          </div>
```

Comment:
- [ ] I would suggest you that technology categories like ("Languages", "Frameworks" and "Skills") should be in a ul, each of them in a li for better semantic HTML. Please implement it